test/Microsoft.AspNetCore.Mvc.Analyzers.Test/TestFiles/AttributesShouldNotBeAppliedToPageModelAnalyzerTest/NoDiagnosticsAreReturned_ForPageHandlersWithNonFilterAttributes.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Runtime.CompilerServices;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace Microsoft.AspNetCore.Mvc.Analyzers.Test
+{
+    public class NoDiagnosticsAreReturned_ForPageHandlersWithNonFilterAttributes : PageModel
+    {
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Curious, what does this do?

Reply:
Nothing in particular. Just random attributes that could be applied to methods.